nemo_skills/inference/eval/hilbench.py

```
@@ -0,0 +1,897 @@
+            if self.cfg.agent_framework == SupportedAgentFrameworks.swe_agent:
+                pred_file = await self._run_swe_agent_hil(data_point, api_base)
+            elif self.cfg.agent_framework == SupportedAgentFrameworks.gold_patch:
+                pred_file = await self._get_gold_patch(data_point)
+            else:
+                raise ValueError(
+                    f"HIL-Bench currently supports agent_framework in {{swe_agent, gold_patch}}, "
+                    f"got {self.cfg.agent_framework}."
+                )
+            with open(pred_file, "r") as f:
+                trajectory_dict = json.loads(f.read())
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)